src/test/java/racingcar/service/RacingCarServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package racingcar.service;
+
+import static org.assertj.core.api.AssertionsForClassTypes.*;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.Test;
+
+import racingcar.model.Car;
+
+class RacingCarServiceTest {
+
+	private RacingCarService racingCarService;
+	private final int expectMaxPosition = 4;
+
+	public void customRacingCars() {
+		Car car1 = new Car("juri", 2);
+		Car car2 = new Car("hunch", 1);
+		Car car3 = new Car("pobi", expectMaxPosition);
+		racingCarService = new RacingCarService(new ArrayList<>(Arrays.asList(car1, car2, car3)));
+	}
+
+	@Test
+	public void 최대값_찾기_테스트() {
+		customRacingCars();
+		assertThat(racingCarService.findMaxPosition()).isEqualTo(expectMaxPosition);
+	}
+
+	@Test
+	public void 우승자_찾기_테스트() {
+		customRacingCars();
+		ArrayList<String> result = new ArrayList<>(Collections.singletonList("pobi"));
```

Comment:
변수의 타입에 구체적인 타입보다는 추상적인 타입을 사용해주세요.
구현 클래스가 바뀔 때 쉽게 교체할 수 있습니다.

```suggestion
		List<String> result = Collections.singletonList("pobi");
```